### ISSUE TITLE
Linux x86_64 "file not found"

### ISSUE THREAD

**Issue by gstalnaker** [KYDronePilot/hdfm#29]
```
> $ sudo curl -sSL 'https://github.com/KYDronePilot/hdfm/releases/download/v2.2.0/hdfm-x86_64-unknown-linux-musl.tgz'                  
Not Found%  
```

**Comment by gstalnaker** [comment]
P.S. Just so you know that the directions on your "facing" page for the Linux download result in that error on my new Kubuntu 23.10 system (zsh shell).. I CAN get the file manually (and have done so) using the [Releases](https://github.com/KYDronePilot/hdfm/releases) link further down the page. No idea why I get that error. I checked the link and it looks good to my eyes.

**Comment by redrob** [comment]
Yeah seems like release (per the README) 2.2.0 is missing. Any update on this?

**Comment by benmmaddog** [comment]
It seems the 2.2.0 version only has the source files , there isn't any OS related files
